.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: tests
+
+on: [pull_request, push]
+
+env:
+  FORCE_COLOR: 1
+  NO_ANDROID: 1
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: ['lts/*']
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install dependencies (Linux)
+      if: runner.os == 'Linux'
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y git libwebkit2gtk-4.1-dev build-essential libc++abi-14-dev libc++-14-dev pkg-config clang-14
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - run: npm i @socketsupply/socket -g
+    - run: ssc --version
```

Comment:
This step is failing in CI due to getting caught up on android deps. Probably want to look into that soon.